### ISSUE TITLE
ImportError: [...] undefined symbol: lmder_

### ISSUE THREAD

**Issue by revoltek** [lofar-astron/PyBDSF#125]
Maybe related to #123 

I installed the last master of BDSF in debian testing, but it crashes with:
```
Traceback (most recent call last):
  File "/home/fdg/scripts/pybdsm_script.py", line 20, in <module>
    import bdsf
  File "/home/fdg/opt/lib/python3.7/site-packages/bdsf-1.9.2-py3.7-linux-x86_64.egg/bdsf/__init__.py", line 21, in <module>
    from .collapse import Op_collapse
  File "/home/fdg/opt/lib/python3.7/site-packages/bdsf-1.9.2-py3.7-linux-x86_64.egg/bdsf/collapse.py", line 11, in <module>
    from . import _cbdsm
ImportError: /home/fdg/opt/lib/python3.7/site-packages/bdsf-1.9.2-py3.7-linux-x86_64.egg/bdsf/_cbdsm.cpython-37m-x86_64-linux-gnu.so: undefined symbol: lmder_
```
These are the linked libs:
```
ldd /home/fdg/opt/lib/python3.7/site-packages/bdsf-1.9.2-py3.7-linux-x86_64.egg/bdsf/_cbdsm.cpython-37m-x86_64-linux-gnu.so
        linux-vdso.so.1 (0x00007ffcdc167000)
        libgfortran.so.5 => /usr/lib/x86_64-linux-gnu/libgfortran.so.5 (0x00007f3396557000)
        libboost_python-py37.so.1.62.0 => /usr/lib/x86_64-linux-gnu/libboost_python-py37.so.1.62.0 (0x00007f339650b000)
        libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f3396332000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f33961ed000)
        libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f33961d3000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f3396013000)
        libquadmath.so.0 => /usr/lib/x86_64-linux-gnu/libquadmath.so.0 (0x00007f3395fc8000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f3396856000)
        libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f3395fc3000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f3395fa2000)
        libutil.so.1 => /lib/x86_64-linux-gnu/libutil.so.1 (0x00007f3395f9d000)
        librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f3395f92000)
```

**Comment by bmerry** [comment]
I also see this in master but if I check out v1.9.2 it is fine. So presumably it is a recent regression.

**Comment by revoltek** [comment]
v1.9.2 crashes with:
`In file included from natgrid/Src/natgrid.c:6:
natgrid/Include/nnmhead.h:43:28: error: ‘_IO_stderr’ undeclared here (not in a function); did you mean ‘stderr’?
   43 | FILE    *fopen(), *filee = _IO_stderr;
      |                            ^~~~~~~~~~
      |                            stderr
error: Command "x86_64-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -Inatgrid/Include -I/usr/lib/python3/dist-packages/numpy/core/include -I/usr/include/python3.7m -c natgrid/Src/natgrid.c -o build/temp.linux-x86_64-3.7/natgrid/Src/natgrid.o -MMD -MF build/temp.linux-x86_64-3.7/natgrid/Src/natgrid.o.d" failed with exit status 1
`

**Comment by darafferty** [comment]
You should be able to fix that error by applying commit 23e4b72e6 (basically just removes the offending lines in nnmhead.h). Unfortunately, I've still no idea about how to solve your original error.

**Comment by gmloose** [comment]
I'm closing this issue, since it's three years old. Feel free reopen it if the problem still exists with the latest release.
